c10/core/CPUAllocator.cpp

```diff
@@ -153,14 +153,18 @@ void SetCPUAllocator(at::Allocator* alloc) {
   SetAllocator(DeviceType::CPU, alloc);
 }
 
-// Global default CPU Allocator
-static DefaultCPUAllocator g_cpu_alloc;
+// #ifndef C10_MOBILE
```

Comment:
Disregard this section for now.  I am intentionally making this path run for all mobile and non-mobile builds to exercise it more with CI.  This will be mobile only.